pubspec.yaml

```diff
@@ -8,11 +8,17 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/stream_channel
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.8.0-dev.15 <3.0.0'
 
 dependencies:
   async: '>=1.11.0 <3.0.0'
 
 dev_dependencies:
   pedantic: ^1.8.0
   test: ^1.2.0
+
+dependency_overrides:
```

Comment:
You might want to add the full list of overrides for the tests, https://gist.githubusercontent.com/jakemac53/23f62a98d6226e80c082264fe9f0e6b3/raw/4a699ae440ee4b1060a05d56011bcfa8d1de701e/overrides.yaml

Reply:
Done